src/NumericalAlgorithms/LinearOperators/Python/PowerMonitors.cpp

```diff
@@ -0,0 +1,34 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "NumericalAlgorithms/LinearOperators/Python/PowerMonitors.hpp"
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "DataStructures/DataVector.hpp"
+#include "NumericalAlgorithms/LinearOperators/PowerMonitors.hpp"
+#include "NumericalAlgorithms/Spectral/Mesh.hpp"
+
+namespace py = pybind11;
+
+namespace PowerMonitors::py_bindings {
+
+namespace {
+template <size_t Dim>
+void bind_power_monitors_impl(py::module& m) {  // NOLINT
+  m.def("power_monitors",
+        py::overload_cast<const DataVector&, const Mesh<Dim>&> (
+          &power_monitors<Dim>),
+        py::arg("input_data_vector"), py::arg("mesh"),
+        py::return_value_policy::copy);
```

Comment:
Do you need this return value policy or can you just delete it?

Reply:
Let me test this again. I remember having trouble when I was making the plots in python.

Reply:
Ok, we can add this back if we encounter problems